openstack/barbican/templates/ingress.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
     kubernetes.io/tls-acme: "true"
     disco: "true"
   {{- end }}
+  {{- include "utils.linkerd.ingress_annotation" . | indent 4 }}
```

Comment:
This will create invalid YAML if `.Values.tlsacme` is not set. I think the template needs restructuring to always add the `annotations` key, i.e. move the `if` from line 15 down a line.